workers/unity/Packages/io.improbable.gdk.testutils/Editor/Tests/Improbable.Gdk.TestUtils.Editor.Tests.asmdef

```diff
@@ -0,0 +1,19 @@
+{
+    "name": "Improbable.Gdk.TestUtils.Editor.Tests",
+    "references": [
+        "Improbable.Gdk.TestUtils.Editor"
+    ],
+    "optionalUnityReferences": [
+        "TestAssemblies"
+    ],
+    "includePlatforms": [
+        "Editor"
+    ],
+    "excludePlatforms": [],
+    "allowUnsafeCode": false,
+    "overrideReferences": false,
+    "precompiledReferences": [],
+    "autoReferenced": true,
```

Comment:
our test assemblies should probably leave this off to speed things up a little

Reply:
Done!